### DISCUSSION TITLE
Fehler in Kapitel 4 Tabelle 4.2

### DISCUSSION THREAD

**Discussion by dt-flo** [TobiasPSP/OReilly#19] [Fragen] [answered]
Hi,

erstmal vorab: großartiges Buch! Freue mich schon auf die nächste Auflage :)

In Tabelle 4.2 findet man folgendes:
![firefox_2023-08-27_15-18-48](https://github.com/TobiasPSP/OReilly/assets/25205082/152b022e-f7f2-4bf0-8505-34797392a323)

Allerdings verhält sich `-Ea Ignore` bei mir anders. "SilentlyContinue" protokolliert den Fehler, aber unterdrückt ihn. "Ignore" verwirft ihn und protokolliert ihn nicht.

Grüße,
Florian

**Comment by dt-flo** [comment]
Ah ich sehe gerade, in Tabelle 4.3 steht es richtig
![firefox_2023-08-27_15-25-30](https://github.com/TobiasPSP/OReilly/assets/25205082/1967d361-af97-4e27-8838-773cc10d92c5)

**Comment by TobiasPSP** [comment, marked as answer]
Vielen Dank für den Hinweis!
